### PR TITLE
activeDeadlineSeconds and commonLabels support

### PR DESCRIPTION
**Pull request by kvanzuijlen**

### CODE REVIEW

**Review by viceice — CHANGES_REQUESTED** [review]
add doc lines to values and update readme

**Comment by kvanzuijlen** [comment]
Will add them to the values.yaml as well

**Comment by kvanzuijlen** [comment]
Thx for your quick response @viceice, hope these changes are sufficient!

**Review by viceice — COMMENTED** [review]
don't require review if it's not fully done to reduce noise.

**Comment by kvanzuijlen** [comment]
@viceice should be fully done right now, or am I still missing something? I don't really see why the lint check is failing.

**Comment by viceice** [comment]
@JamieMagee do you know the error?

File: charts/renovate/templates/_helpers.tpl [hunk 1]
```diff
@@ -49,6 +49,7 @@ Selector labels
 {{- define "renovate.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "renovate.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+{{ .Values.global.commonLabels | toYaml }}
```
[the remark this answers is not in the document]
Comment (kvanzuijlen) [reply charts/renovate/templates/_helpers.tpl]: I was just thinking the same thing, added them and the issue got solved (locally) after first reproducing the issue. Thx for your help @viceice

**Review by viceice — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**